### ISSUE TITLE
brace-style

### ISSUE THREAD

**Issue by eliascodes** [dwyl/goodparts#217]
http://eslint.org/docs/rules/brace-style

```
["error", "1tbs", { "allowSingleLine": true }]
```

Use 1tbs by default, allow single line statements, although should only use them in v. simple cases

**Comment by nelsonic** [comment]
At present the rule is requiring the else statement to start on the _same_ line as the closing brace for the `if`:
```
if (condition) {
  // do things
} else {
  // do alternative things
}
```
I _prefer_ the `else` on the next line for legibility ...
```
if (condition) {
  // do things
}
else {
  // do alternative things
}
```
Anyone have thoughts on this?

**Comment by nelsonic** [comment]
@eliascodes I _very much_ prefer to use the **`Stroustrup`** brace style 
putting the `else` block start on the same level of indentation as the `if` as shown in the second example above makes the code more readable **IMO**...
If nobody has any _objections_ I would like to enforce the `"stroustrup"` as our `brace-style` ...

**Comment by eliascodes** [comment]
@nelsonic I have no particular objections, might want to ask someone who uses `goodparts` on a daily basis though. 

Also be aware that JSLint takes the opposite view:
![screen shot 2016-11-23 at 11 11 22](https://cloud.githubusercontent.com/assets/12719553/20559697/b4a5b746-b16d-11e6-91fb-c9f64af214d0.png)
![screen shot 2016-11-23 at 11 11 07](https://cloud.githubusercontent.com/assets/12719553/20559699/b76db5e6-b16d-11e6-8db4-b0b0454cdc8c.png)

**Comment by nelsonic** [comment]
@eliascodes _great_ point! 🔍 
Yeah, http://jslint.com/ expects `"double-quotes"` for `String` and `four-spaces` indentation ... 😕 
We aren't going to "_conform_" to those JSLint "_stylistic suggestions_"... _right_...? 🤔

**Comment by eliascodes** [comment]
@nelsonic not necessarily, I was just pointing it out. Like I said, I'm happy either way. You can make arguments for legibility/comprehension on both sides I think.
